src/actions.js

```diff
@@ -35,3 +35,22 @@ export function setActiveSearch(searchText) {
     searchText
   };
 }
+
+export const SHOW_MODAL = 'SHOW_MODAL';
+
+export function showModal(user) {
```

Comment:
I would make these a bit more explicit `showUserModal` `SHOW_USER_MODAL` etc. It's fine for the moment but would save time in refactoring when additional modals are introduced.